### ISSUE TITLE
Crashes on Windows10 x64

### ISSUE THREAD

**Issue by KaijuKoder** [nathanfranke/gdextension#5]
Tried running this on Windows 10 x64. Steps 1 and 2 work fine in the README, but when I get to Step 3 "Import, edit, and play `project/` using Godot Engine 4+", Godot 4 lets me import the project but when I try and open it, Godot4 crashes without any error message.

```
c:\test\gdextension>c:\test\Godot\Godot_v4.0-beta10_win64.exe

c:\test\gdextension>Godot Engine v4.0.beta10.official.d0398f62f - https://godotengine.org
OpenGL Renderer: NVIDIA GeForce RTX 3060 Ti/PCIe/SSE2

Editing project: c:/test/gdextension/project

c:\test\gdextension>
```

Given Godot 4 is still in flux, don't think it's worth trying to fix this. Letting you know in case others try go the GDExtension route. It seems no better than GDNative which came before it, and just as fragile. Maybe Modules are a better way to go.

**Comment by frankhale** [comment]
Very likely the same thing that happened to me a while ago. Have a look here. It looks like `godot-cpp` needs to be updated. I went looking for the specific commit (in `godot-cpp`) that would go with the version of `godot` that you  have in your message above but I couldn't find it. 

https://github.com/nathanfranke/gdextension/issues/1

**Comment by KaijuKoder** [comment]
Thanks @frankhale. I tried your suggestion, but still couldn't get it working. I'm probably doing something wrong with git or the versions, but the instructions are unclear, even on Godot's own documentation website.

However I did find Custom Modules much, much easier, though even they have changed in Version 4: https://docs.godotengine.org/en/latest/contributing/development/core_and_modules/custom_modules_in_cpp.html
